modules/nf-core/parabricks/fq2bam/tests/main.nf.test

```diff
@@ -0,0 +1,174 @@
+nextflow_process {
+
+    name "Test Process PARABRICKS_FQ2BAM"
+    script "../main.nf"
+    process "PARABRICKS_FQ2BAM"
+    config "./nextflow.config"
+
+    tag "bwa/index"
+    tag "modules"
+    tag "parabricks/fq2bam"
+    tag "modules_nfcore"
+    tag "parabricks"
+
+    setup {
+        run("BWA_INDEX") {
+            script "../../../bwa/index/main.nf"
+            process {
+                """
+                input[0] = Channel.of([
+                    [ id:'test' ], // meta map
+                    file('https://github.com/nf-core/test-datasets/raw/methylseq/reference/genome.fa', checkIfExists: true)
+                ])
+                """
+            }
+        }
+
+        run("BWA_INDEX", alias: 'BWA_INDEX_PE') {
+            script "../../../bwa/index/main.nf"
```

Comment:
why a separate setup for PE ? it cna be reused right ?

Reply:
Because PE is using a different reference!

Reply:
of course....plz ignore :)